README.md

```diff
@@ -45,6 +45,7 @@ This extension has the following configuration settings:
 * `pgFormatter.noComment` - Remove any comments (Default: `false`)
 * `pgFormatter.noGrouping` - Add a newline between statements in transaction regroupement (Default: `false`)
 * `pgFormatter.noExtraLine` - Do not add an extra empty line at end of the output (Default: `false`)
+* `pgFormatter.keepNewline` - Preserve empty line in plpgsql code (Default: `false`)
```

Comment:
To take advantage of the new option you added, we should bump the version of psqlformat to the latest version [here](https://github.com/bradymholt/vscode-pgFormatter/blob/main/package.json#L212).  I released a new version so the latest should have your change.